ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/custom-transit-switch-masquerade-subnet/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ovn-conf-internal-masquerade-subnet
```

Comment:
is there any way to make sure `- ref: ovn-conf-internal-masquerade-subnet` should be trigeered as a part of pre-install and following steps should be tested once cluster comes up ?


```
  - ref: ovn-defaultnetwork-internal-masquerade-subnet
  - ref: ovn-defaultnetwork-internal-join-subnet
  - ref: ovn-defaultnetwork-internal-transit-switch-subnet

```

Tried similar way with `pre` and `test` keys as in [dev pr](https://github.com/openshift/release/pull/51970/files#diff-525f4a13f49efad954cc8bd4a249d43d8b610e7221d5da43737d324b6e165147) but it doesn't take `pre` and `test` keys

Reply:
I think current chain/ref order can meet your test requirement. Chain `ipi-install-vsphere` will trigger the cluster installation.

And step `enable-qe-catalogsource` is required after installation for e2e test, need to add at the end along with another step `cucushift-installer-check`
```
  - ref: enable-qe-catalogsource
  - chain: cucushift-installer-check
```

Reply:
Sounds good @jinyunma Thanks

Reply:
and   `- ref: idp-htpasswd`

Reply:
@zhaozhanqi this step already have it [step](https://github.com/openshift/release/blob/587a4f2b338371e7fe1b0bb93de972c2a2f53634/ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml#L5)